packages/expo-image/src/Image.types.ts

```diff
@@ -62,6 +62,17 @@ export type ImageStyle = RNImageStyle;
  */
 export type ImageContentFit = 'cover' | 'contain' | 'fill' | 'none' | 'scale-down';
 
+/**
+ * Determines downscale dimensions for the image.
+ * When only `height` is provided, the `width` will be calculated automatically to preserve the aspect ratio, similarly
+ * when only `width` is provided the `height` will be calculated automatically to preserve the aspect ratio.
+ * When both are provided `width` value will be ignored and the dimensions will be calculated based on the height value.
+ * */
```

Comment:
```suggestion
 * Determines downscale dimensions for the image.
 * When only `height` is provided, the `width` will be calculated automatically to preserve the aspect ratio. Similarly
 * when only `width` is provided, the `height` will be calculated automatically to preserve the aspect ratio.
 * When both are provided, the `width` value will be ignored, and the dimensions will be calculated based on the `height` value.
 * */
```

- We can break split the sentence here
- Formatting and consistency nit
- Missing inline code highlight for height in the last sentence